tests/integration/test_label.py

```diff
@@ -30,20 +29,14 @@ def test_labels(label_pack):
     assert list(data_row.labels()) == []
 
 
-def test_label_export(client, label_pack):
-    project, dataset, data_row, label = label_pack
-    #Project has to be setup for export to be possible
-    editor = list(
-        client.get_labeling_frontends(
-            where=LabelingFrontend.name == "editor"))[0]
-    empty_ontology = {"tools": [], "classifications": []}
-    project.setup(editor, empty_ontology)
-    project.create_label(data_row=data_row, label="export_label")
+def test_label_export(client, configured_project_with_label):
+    project, label_id = configured_project_with_label
+
     exported_labels_url = project.export_labels()
     assert exported_labels_url is not None
     exported_labels = requests.get(exported_labels_url)
-    labels = [example['Label'] for example in exported_labels.json()]
-    #assert 'export_label' in labels
+    labels = [example['ID'] for example in exported_labels.json()]
```

Comment:
since we aren't adding in "export_label" specifically in this test anymore, thought it might be more appropriate to test against the ID instead